topics/fair/tutorials/zenodo-in-galaxy/tutorial.md

```diff
@@ -0,0 +1,157 @@
+> >
+> {: .comment}
+> 
+{: .hands_on}
+
+## How to connect Zenodo to Galaxy 25.0
+
+The new `Manage Your Repositories` section is available under `User → Preferences`. Here’s how it works:
+
+> <hands-on-title>Connecting to Galaxy</hands-on-title>
+> 
+> 1. Navigate to `User → Preferences → Manage Your Repositories`. If you haven't set up any integrations yet, you'll see an empty list.
+> ![Manage Your Remote File Sources](../../images/1-empty-file-sources-list.png)
+>
+> 2. Click `+ Create` to configure a new integration. You'll see a list of available integrations, including `S3, Dropbox, InvenioRDM, Zenodo`, and more (depending on your Galaxy server).
+> ![Create New Integration](../../images/2-create-new.png)
+> 
+> 3. Select `Zenodo` to configure it and enter your **credentials or relevant connection details**. (This will be the access token created in the previous section if using the Zenodo sandbox.)
+> ![Configure Zenodo](../../images/3-zenodo-template-filled.png)
+> 
+> 4. Once set up, the Zenodo integration, in this case named `My Zenodo`, will appear in the list where you can manage or delete it.
+> ![Zenodo Integration](../../images/4-instance-created-zenodo.png)
+> 
+> 5. The Zenodo integration, along with any other integrations that you have set up, will appear first when browsing import/export locations. You can find them by clicking on the `Upload` button in the activity bar.
+> ![Browse Remote Files](../../images/5-upload-import-zenodo.png)
+{: .hands_on}
+
+
+> <details-title>Do you want to add a different repository to Galaxy?</details-title>
+> 
+> There are a number of additional repository types that you can connect to Galaxy and use to import and export data.
+>
+> {% snippet faqs/galaxy/manage_your_repositories.md %}
+>
+{: .details}
+
+# Importing records and files into Galaxy
+
+Once you have connect Zenodo to Galaxy, you will be able to import records and files stored in Zenodo into Galaxy.
+
+> <hands-on-title>Importing Files</hands-on-title>
+> 
+> 1. Open the Upload tool, and select `Choose remote files`
+> ![Upload Tool](../../images/upload-tool.png)
+>
+> 2. Search for the Zenodo instance. Remember that this will only appear if the Zenodo plugin is configured to connect to Zenodo in your Galaxy instance.
+> ![Choose Remote Files](../../images/choose-remote-files-zenodo.png)
+>
+{: .hands_on}
+
+Once you have selected Zenodo, you can browse public records and import them into your Galaxy history. You can choose to import the full record or individual files in the same way you would import files from any other remote source.
+
+# Exporting your Galaxy history
+
+You can export your history directly to Zenodo. A benefit of publishing your history on Zenodo is that Zenodo will generate a DOI for your history. You can then include this DOI in your publication and it can be used by other researchers to cite your work.
```

Comment:
this is great, it'll directly answer a user question I keep receiving, now I have something to link them to!